front/src/components/organisms/CategoriesList.tsx

```diff
@@ -62,6 +62,7 @@ const categoryImg: Record<string, string> = {
   Python: icons.python, // TODO: Add Python Icon
   React: icons.react, // TODO: Add React Icon
   Spring: icons.spring, // TODO: Add Spring Icon
+  Laravel: icons.laravel, // TODO: Add Spring Icon
```

Comment:
fix TODO codeSmell. Maybe if TODO´s associated tasks are completed, can be removed

Reply:
The TODOs are not completed yet: the final version will provide server icons instead of front assets and this conversion will be unnecessary